pymc3/stats.py

```diff
@@ -342,7 +342,7 @@ def bpic(trace, model=None):
 
 
 def compare(traces, models, ic='WAIC', method='stacking', b_samples=1000,
-            alpha=1, seed=None):
+            alpha=1, seed=None, round_to=2):
```

Comment:
For all these measures, a difference of that scale is not relevant. Even 2 is more precision than required.